frontend/public/components/utils/line-buffer.ts

```diff
@@ -0,0 +1,49 @@
+import * as _ from 'lodash';
+
+export const LINE_PATTERN = /^.*(\r\n|\n|$)/gm;
```

Comment:
`.*\n` should also match `.*\r\n`. If I'm not wrong, this can be `/^.*(\n|$)/gm`